velox/common/memory/MemoryAllocator.cpp

```diff
@@ -324,22 +313,14 @@ bool MallocAllocator::allocateContiguousImpl(
   numAllocated_.fetch_add(numPages);
   numMapped_.fetch_add(numNeededPages + numContiguousCollateralPages);
 
-  if (TestValue::enabled()) {
-    bool injectMmapError = false;
-    TestValue::adjust(
-        "facebook::velox::memory::MallocAllocator::allocateContiguousImpl",
-        &injectMmapError);
-    if (injectMmapError) {
-      return false;
-    }
-  }
   void* data = ::mmap(
       nullptr,
       numPages * AllocationTraits::kPageSize,
       PROT_READ | PROT_WRITE,
       MAP_PRIVATE | MAP_ANONYMOUS,
       -1,
       0);
+  // TODO: add to handle ::mmap error MAP_FAILED.
```

Comment:
typos? "TODO Add handling of MAP_FAILED"?

Reply:
Thanks!